### PR TITLE
Updated some quests

### PR DESCRIPTION
**Pull request by zcarp7220**

### CODE REVIEW

File: config/ftbquests/quests/chapters/early_game.snbt [hunk 1]
```diff
@@ -1333,7 +1333,7 @@
...
+				"The tower we'll be building at first is 3x3x5 (LxWxH), with a hollow center. The &6Fluid Input Hatch&r &2can go anywhere on the bottom layer.&r. The &6controller&r goes in the middle of an outer edge of the bottom layer. The \\& &6Energy Input Hatch&r can go wherever you can fit it on the bottom layer. The rest of the 3x3 must be filled with &6Clean Stainless Steel Casings&r.&2 You also need a Maintenance Hatch, which can go anywhere on the structure.&r Each level above that is a ring of 7 Clean Stainless Steel Casings, one &6Fluid Output&r and nothing in the center, except for the topmost layer which needs an eighth casing to fill in the center, capping off the top. "
```
Comment (LetterN) [inline config/ftbquests/quests/chapters/early_game.snbt]:
```suggestion
				"The tower we'll be building at first is 3x3x5 (LxWxH), with a hollow center. The &6Fluid Input Hatch&r &2can go anywhere on the bottom layer.&r. The &6controller&r goes in the middle of an outer edge of the bottom layer. The &6Energy Input Hatch&r can go wherever you can fit it on the bottom layer. The rest of the 3x3 must be filled with &6Clean Stainless Steel Casings&r.&2 You also need a Maintenance Hatch, which can go anywhere on the structure.&r Each level above that is a ring of 7 Clean Stainless Steel Casings, one &6Fluid Output&r and nothing in the center, except for the topmost layer which needs an eighth casing to fill in the center, capping off the top. "
```

**Review by LetterN — COMMENTED** [review]
other than that pr looks fine

**Comment by ThePansmith** [comment]
Loot superfab quest already exists

**Comment by zcarp7220** [comment]
fixed it

**Review by LetterN — APPROVED** [review]

**Pull request merged**